packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -249,6 +249,43 @@ const Conversation = WebexPlugin.extend({
     return activity;
   },
 
+
+  /**
+  * Gets an array of activities with an array of activity URLS
+  * @param {Array} activties
```

Comment:
Spelling of `activities`

Reply:
I fixed that and it keeps being wrong. I'll try again